lib/rules/font-family-generic-fallback/index.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const declarationValueIndex = require("../../utils/declarationValueIndex");
+const findFontFamily = require("../../utils/findFontFamily");
+const keywordSets = require("../../reference/keywordSets");
+const report = require("../../utils/report");
+const ruleMessages = require("../../utils/ruleMessages");
+const validateOptions = require("../../utils/validateOptions");
+
+const ruleName = "font-family-generic-fallback";
+
+const messages = ruleMessages(ruleName, {
+  generic: () => "Missing generic font family, e.g. serif"
+});
+
+const isFamilyNameKeyword = node =>
+  !node.quote && keywordSets.fontFamilyKeywords.has(node.value.toLowerCase());
+
```

Comment:
Does `!node.quote` mean anything in quotes will NOT be seen as a generic family name? I would have assumed that `font-family: Times, "serif"`, is perfectly valid and should pass, although it is slightly weird.

Reply:
According to [the spec](https://drafts.csswg.org/css-fonts/#font-family-prop), generic font families must be unquoted:

> The following generic family keywords are defined: ‘serif’, ‘sans-serif’, ‘cursive’, ‘fantasy’, and ‘monospace’. These keywords can be used as a general fallback mechanism when an author's desired font choices are not available. As keywords, they must not be quoted. Authors are encouraged to append a generic font family as a last alternative for improved robustness.

So, the code as it is LGTM.

Reply:
I stand corrected ... agreed.